src/StatsTools/growth_witness.jl

```diff
@@ -36,14 +35,18 @@ function growth_witness(shift::AbstractArray, norm::AbstractArray, dt, b; kwargs
 end
 """
     growth_witness(df::DataFrame, [b]; shift=:shift, norm=:norm, dτ=df.dτ[end], skip=0)
+
 Calculate the growth witness directly from a `DataFrame` returned by
 [`lomc!`](@ref Main.lomc!). The keyword arguments `shift` and `norm`
```

Comment:
The reference to `lomc!` is obsolete.
```suggestion
Calculate the growth witness directly from the result (`DataFrame` or [`PMCSimulation`](@ref)) of `solve`ing a 
[`ProjectorMonteCarloProblem`](@ref Main.ProjectorMonteCarloProblem). The keyword arguments `shift` and `norm`
```